alws/crud/build_node.py

```diff
@@ -134,11 +144,48 @@ async def __process_rpms(pulp_client: PulpClient, task_id: int, task_arch: str,
                     href=href
                 )
             )
+    if exceptions:
+        raise ArtifactConversionError(
+            'Cannot put RPM packages into Pulp storage: %s',
+            '\n'.join([str(e) for e in exceptions]))
+    for repo_href, packages in repo_packages_mapping.items():
+        try:
+            await pulp_client.modify_repository(repo_href, add=packages)
+        except Exception as e:
+            logging.error('Cannot add RPM packages to the repository: %s',
+                          str(e))
+            exceptions.append(e)
+        try:
+            await pulp_client.create_rpm_publication(repo_href)
```

Comment:
Build rpm repos triggers rpm publication after adding/removing content, so may be we don't need this?

Reply:
That's exactly what I've got rid of. Slava pointed out that such approach triggers 3 tasks per each package in pulp (3*N) while modernized approach is N + 2, where N is number of packages and 2 - adding content to repository + making publication. In order to achieve it we first do just artifacts -> RPMs conversion, then add RPMs to repository, then manually trigger publication

Reply:
Okay, so may be we need to turn off autopublish for rpm repos?

Reply:
Agreed with @anfimovdm, why we need to remove autopublish from repos? It's actually +1 request for every repository, which makes whole thing slower